main.tf

```diff
@@ -144,5 +147,6 @@ module "service_task" {
   vpc_id                             = var.vpc_id
   wait_for_steady_state              = var.wait_for_steady_state
 
-  context = module.this.context
+  label_orders = var.label_orders
```

Comment:
does this actually work? I wonder because the label_orders of this module do contain more variables than the one from the ecs-alb-service-task module